backend/spacewalk-backend.changes

```diff
@@ -1,6 +1,7 @@
 -------------------------------------------------------------------
 Fri Oct 26 10:06:41 CEST 2018 - jgonzalez@suse.com
 
+- Hide Python traceback and show only error message (bsc#1110427)
```

Comment:
Wait! This changelog entry is wrong, it should be added at the top of the file. Maybe this was caused by git while cherrypicking